Frontend/src/locales/de.json

```diff
@@ -75,9 +75,22 @@
   },
   "photos":  {
     "myPhotos": "Meine Photos",
-    "friendsPhotos": "Photos deiner Freunde"
+    "descriptionMyPhotos": "Hier kannst Du Dir alle Photos ansehen, die Du zu den Challenges der Photowalks hochgeladen hast. Bilder, die Deinen Freunden besonders gut gefallen haben viele Herzmarker.",
+    "friendsPhotos": "Photos meiner Freunde",
+    "descriptionFriendsPhotos":
+      "Hier kannst Du Dir alle Fotos ansehen, die Deine Freunde zu den Challenges der Photowalks hochgeladen haben. Erkennst Du einige der Motive wieder? Gefällt Dir ein Bild besonders gut, dann zeige es Deinen Freunden indem du auf das Herz klickst."
   },
   "photowalks": {
-    "chooseWalk" : "Wähle einen Photowalk aus"
+    "chooseWalk" : "Wähle einen Photowalk aus",
+    "startWalk": "Walk starten"
+  },
+  "mobileView": {
+    "mapTab": "Karte",
+    "challengesTab": "Challenges & Info",
+    "photosTab": "Photos"
+  },
+  "landing": {
+    "headline":"Lust auf einen Spaziergang?",
+    "pitch": "Hättest du Lust mal wieder an die frische Luft zu gehen, findest alleine spazieren gehen aber langweilig? Würdest du gerne mal wieder mit deinen Freunden zusammen eine Fotorally starten und schauen, wer die ausgefallensten Fotos macht? Mit PhotoWalk ist das selbst in schwierige Zeiten mit social distancing kein Problem! Gehe auf einen von 3 kuratierten Spaziergängen in der schönsten Stadt des Nordens. Nimm deine Kamera mit, lichte ein paar tolle Motive ab und messe dich in Photochallenges mit deinen Freunden. Wie interpretieren deine Freunde die von uns gestellten Challenges? Was kannst du Neues auf deinem Spaziergang entdecken? Jetzt registrieren und herausfinden."
```

Comment:
Hättest Du Lust mal wieder an die frische Luft zu gehen, findest alleine spazieren gehen aber langweilig? Würdest Du gerne mal wieder mit Deinem Freundeskreis zusammen eine Fotorally starten und schauen, wer die ausgefallensten Fotos macht? Mit PhotoWalk ist das selbst in schwierigen Zeiten mit Social Distancing kein Problem! Gehe auf einen von 3 kuratierten Spaziergängen in der schönsten Stadt des Nordens. Nimm Deine Kamera mit, lichte ein paar tolle Motive ab und messe Dich in Photochallenges mit deinem Freundeskreis. Wie interpretiert Dein Freundeskreis die von uns gestellten Challenges? Was kannst Du Neues auf Deinem Spaziergang entdecken? Jetzt registrieren und herausfinden.